Menu/Menu.less

```
@@ -1,30 +1,73 @@
+.main-content {
+  display: flex;
+}
+
 .menu-button {
   position: absolute;
   left: 20px;
   cursor: pointer;
 }
 
 .menu {
-  display: none;
-  position: fixed;
-  top: 75px;
-  left: 0;
-  width: 350px;
+  // display: none;
+  // position: fixed;
+  margin-top: 75px;
+  // left: 0;
+  width: 0px;
   height: 100vh; /* This give the height 100% of the screen or, '100% of View Height' */
-  background-color: #81C784;
-  border: 1px solid #004D40;
-  border-left: none;
-  border-bottom: none;
-  z-index: 2; /* This set the layer level of the element, it has precendence over initial level of 1*/
-}
+  background-color: white;
+  // border: 1px solid #004d40;
+  // border-left: none;
+  // border-bottom: none;
+  // z-index: 2; /* This set the layer level of the element, it has precendence over initial level of 1*/
 
-.menu ul li{
-  list-style-type: none;
-  font-size: 32px;
-  padding: 30px 0;
-  text-align: center;
+  ul {
+    list-style: none;
+    opacity: 0;
```

```suggestion
    overflow: hidden;
```
This is just a suggestion, what you have is working fine. If you set the ul to overflow: hidden, it will get properly hidden when the menu's width is 0, so you don't have to do the extra opacity tween. I just wanted to point it out in case it helps you in the future, everything you have here works just fine as well.